changelog.md

```diff
@@ -30,6 +30,7 @@
 * [1966](https://github.com/zeta-chain/node/pull/1966) - move TSS vote message from crosschain to observer
 * [1853](https://github.com/zeta-chain/node/pull/1853) - refactor vote inbound tx and vote outbound tx
 * [2001](https://github.com/zeta-chain/node/pull/2001) - replace broadcast mode block with sync and remove fungible params
+* [2014](https://github.com/zeta-chain/node/pull/2014) - remove params module
```

Comment:
Will be after the next release
Maybe we can already add another unrelease section in the top to not forget about putting this one here